samples/net/secure_mqtt_sensor_actuator/src/device.c

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (c) 2024 Analog Devices, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(app_device, LOG_LEVEL_DBG);
+
+#include <zephyr/kernel.h>
+#include <zephyr/drivers/sensor.h>
+#include <zephyr/drivers/led.h>
+#include <zephyr/random/random.h>
+
+#include "device.h"
+
+#define SENSOR_CHAN     SENSOR_CHAN_AMBIENT_TEMP
+#define SENSOR_UNIT     "Celsius"
+
+/* Devices */
+static const struct device *sensor = DEVICE_DT_GET_OR_NULL(DT_ALIAS(ambient_temp0));
+static const struct device *leds = DEVICE_DT_GET_ONE(gpio_leds);
```

Comment:
Would need to use `DEVICE_DT_GET_OR_NULL()` for leds as well, currently it won't build for emulated platforms.

Reply:
Done